client/src/components/ViewProfile.js

```diff
@@ -35,150 +22,197 @@ const ViewProfile = () => {
     }
   }, [profileId, delProfile, history]);
 
+  const confirmDelete = deleteOrNot => {
```

Comment:
`deleteOrNot` feels like a weird name .. not sure what true would mean in this case. Maybe `deleteConfirmed` is clearer.